test/unit/specs/components/v-checkbox.spec.js

```diff
@@ -37,14 +36,6 @@ const TestWrapperStringLabel = ({
   })
 }
 
-// eslint-disable-next-line vue/one-component-per-file
```

Comment:
Why remove this test case? It was testing that you can put another component inside the checkbox component slot.

Reply:
It wasn't testing conditional rendering by component behaviour, this is more of a Vue feature, the slots, so I think we can assume it will work.